src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    protected Color color;
+}
```

Comment:
Need to create constructor and init color

Reply:
all work done I don't need to do this!

Reply:
Create a constructor for color.
If you disagree with some comment - discuss it in MA chat, not in PR